packages/form-core/test/validate/Validator.test.js

```diff
@@ -103,15 +116,19 @@ describe('Validator', () => {
   it('fires "param-changed" event on param change', async () => {
     const vali = new Validator('foo');
     const cb = sinon.spy(() => {});
-    vali.addEventListener('param-changed', cb);
+    if (vali.addEventListener) {
```

Comment:
Is this needed because TS doesn't know about existance of `addEventListener` on prototype?

In that case we could better add (gets rid of tech debt and these workarounds):
```
EventTargetMixin extends superclass {
   addEventListener(...){...}
}
```

Reply:
Ah yeah I suggested the same in a previous reply to your comments :P I think a mixin is the most elegant solution, and although I try not to make hasty abstractions, here it makes sense for ease of typing.. but perhaps we can do this in a separate MR for keeping the scope on form-core. 

We will probably do it in lion core? Then we release everything, so also good idea to sync that MR with this MR, since this one will release all form related packages.